.lgtm.yml

```diff
@@ -0,0 +1,4 @@
+---
+queries:
+  # https://lgtm.com/rules/6770079/
+  - exclude: py/unused-import
```

Comment:
in https://github.com/bids-standard/bids-specification/pull/865#issuecomment-918180786 I meant something like this:

```suggestion
---
# This file contains configuration for the LGTM tool: https://lgtm.com/
# The bids-specification repository is continuously scanned by the LGTM tool
# for any security and/or code vulnerabilities.
# You can find the report here: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
queries:
  # https://lgtm.com/rules/6770079/
  - exclude: py/unused-import
```

Where XXXXXXX is to be filled ;-)

Reply:
I'm not sure the link to the specific issue is a good idea. It's quite long:
https://lgtm.com/projects/g/bids-standard/bids-specification/snapshot/3d22fdf48aeb59b3203b3a31f291b375d1dbd17f/files/pdf_build_src/process_markdowns.py#x2d0047ad44974839:1

Reply:
@DimitriPapadopoulos the documentation I propose is intended for  contributors/users who open this file and wonder what it is about: It's not a documentation about the specific queries. So I think XXXXXXXXXXXXX should be:  https://lgtm.com/projects/g/bids-standard/bids-specification/

Reply:
Ah, my wrong. I have been buried in these long URLs lately and had forgotten the base URL is very simple. Hope it's OK this time 😰